.circleci/test-deploy.yml

```diff
@@ -284,6 +284,13 @@ workflows:
           ignore-rules: DL4005,DL3008,DL3009,DL3015,DL3059
           trusted-registries: docker.io,my-company.com:5000
           dockerfiles: test.Dockerfile:test2.Dockerfile
+          failure-threshold: none
+          filters: *filters
+      - test-hadolint-install:
```

Comment:
is this intended to be a duplicate? either add an alias or remove one. `test-deploy` is not running because of this